templates/_formhelpers.html

```diff
@@ -1,9 +1,13 @@
-{% macro render_field(field, label_visible=true) -%}
+{% macro render_field(field, label_visible=true, disabled=False) -%}
      <div class="form-group {% if field.errors %}has-error{% endif %} {{ kwargs.pop('class_', '') }}">
         {% if field.type != 'HiddenField' and field.type != 'CSRFTokenField' and label_visible and field.name != 'id' and field.name != 'form_name' %}
             <label for="{{ field.id }}" class="control-label">{{ field.label }}</label>
         {% endif %}
-        {{ field(class_='form-control', **kwargs) }}
+        {% if disabled %}
+            {{ field(disabled=True, class_='form-control', **kwargs) }}
+        {% else %}
+            {{ field(class_='form-control', **kwargs) }}
+        {% endif %}
```

Comment:
Can't you simplify this block to just: 
```
{{ field(disabled=disabled, class_='form-control', **kwargs) }}
```

Reply:
I see now why you did it that way. Presence of `disabled` attribute makes the element disabled regardless of whether you make it true or false.